### DISCUSSION TITLE
THANK YOU!

### DISCUSSION THREAD

**Discussion by kkurkowski** [LedgerSync/ledger_sync#290] [General]
Thank you for the Ruby how-to post https://www.ledgersync.dev/posts/netsuite-authentication/
I spent two days running into the same walls mentioned in the post. 
I am new to Ruby, so your step-by-step explanation was perfect and very easy to follow. I had my own NetSuite OAuth built out in minutes. Thanks again!

**Comment by ryanwjackson** [comment]
@kkurkowski, we're glad it helped! Please let us know if you have any other questions.
